spec/system/data_migration/baldwin_form_submission_spec.rb

```diff
@@ -64,6 +64,8 @@
       # This will allow us to evolve our local datacite standards and test our records against them.
       datacite = PDCSerialization::Datacite.new_from_work(baldwin_work)
       expect(datacite.valid?).to eq true
+
+      export_spec_data("baldwin.json", baldwin_work.to_json)
```

Comment:
Notice that by default the call to `export_spec_data` does nothing, it only exports the data if `ENV["DATA_MIGRATION"] == "true"`